.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: go
 go:
-  #- 1.5  go test arguments are different in 1.5 :-(
-  - 1.6
```

Comment:
I think 1.6 does not work with the latest release of OSX.  K8s requires 1.7 now, and I assume that we will be building the release with 1.7.  What is the purpose to test it?  I can add comment that we are not longer testing 1.6.